main.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import sys
+from time import sleep
+
+from thespian.actors import ActorAddress, ActorSystem
+
+from src.utils.address_book import AddressBook
+from src.utils.cli import CLI
+from src.utils.constants import ActorNames, Requests
+
+
+def main():
+    """
+    Starts the Bellboy system by configuring logging, creating an ActorSystem,
+    creating the LeadActor, and asking it to START.
+
+    The lead actor loads all other required actors. The system can be
+    halted with a keyboard interrupt (CTRL+C)
+    """
+
+    log = logging.getLogger("Bellboy")
+    log.info("Starting the Bellboy system")
+
+    # Parse arguments and configure logging.
+    cli = CLI()
+    cli.setup(sys.argv)
+
+    # Import actors here to instantiate logging system correctly.
+    from src.actors.lcd import LiquidCrystalActor
```

Comment:
can we move all the "src" folders into the "bellboy" folder? so instead of import src.actors.lcd its simply import actors.lcd.

Reply:
Yeah, looks better too. This was a temporary measure so merging would be easier.

Reply:
alright sounds good, nice